### DISCUSSION TITLE
Disk С sata/m2

### DISCUSSION THREAD

**Discussion by Movet-on** [mathoudebine/turing-smart-screen-python#39] [Original Windows app (USBMonitor / ExtendScreen)]
Good afternoon. There is a problem because of which the C drive is identified by the first SATA drive. But I have the main drive C on M2. As a result, the temperature and volume of the disk are shown completely unnecessary.
Sorry, I use a translator

**Comment by mathoudebine** [comment]
Hello @Admosx 
Unfortunately, it seems USBMonitor does not support NVMe disks like M2 SSD.
I don't know for sure because l'm not the developer of USBMonitor.exe
You can ask the seller of your screen about this issue, or ask on the official forum here: http://discuz.turzx.com/
